main.go

```diff
@@ -5,30 +5,64 @@ import (
 	"fmt"
 	"log"
 	"os"
+	"path/filepath"
 	"strings"
 )
 
 var usage = `
-Usage:	%[1]s PLATFORM [PLATFORM ...]
+Usage:	%[1]s [run] PLATFORM [PLATFORM ...]
+
+Benchmark one or more platforms.
 
 Examples:
 %[1]s platform/python/django
-%[1]s platform/javascript/express
+%[1]s run platform/javascript/express
+
+Usage:	%[1]s report RESULT [RESULT ...]
+
+Print an HTML report summarizing the results of one or more benchmark runs.
+Reports are automatically created after a successful benchmark run.
+This subcommand allows re-generating a report from benchmark result data on demand.
+
+Examples:
+%[1]s result/python/django/20210818-082527-tbnfsga
+%[1]s result/python/django/20210818-*
 `
 
+func printUsage() {
+	fmt.Fprintf(os.Stderr, strings.TrimSpace(usage), filepath.Base(os.Args[0]))
+	fmt.Fprintln(os.Stderr)
+}
+
 func main() {
 	flag.Parse()
 	if len(flag.Args()) < 1 {
-		fmt.Fprintf(os.Stderr, strings.TrimSpace(usage), os.Args[0])
-		fmt.Fprintln(os.Stderr)
+		printUsage()
 		os.Exit(2)
 	}
 	defer func() {
 		if err := recover(); err != nil {
 			log.Fatal(err)
 		}
 	}()
-	for _, platform := range flag.Args() {
-		Benchmark(BenchmarkConfigFromPlatform(platform))
+	switch args := flag.Args(); args[0] {
+	case "report":
+		args = args[1:]
+		if len(args) == 0 {
+			printUsage()
+			os.Exit(2)
+		}
+		Report(args[1:])
```

Comment:
```suggestion
		Report(args)
```

Fixed locally, will update soon.